### ISSUE TITLE
[Bug]: key/value textobject not working properly for joined assignments

### ISSUE THREAD

**Issue by intrntbrn** [chrisgrieser/nvim-various-textobjs#49]
### Bug Description

The following table definition consists of 3  key value assignments

`{ k1 = "v1", k2 = "v2", k3 = "v3" }`

but is interpreted as a single assignment of `k1`.

### Reproduction & Sample Text

before: `{ k1 = "v1", k2 = "v2", k3 = "v3" }`

move cursor to `"v3"` and perform `civ`

after:   `{ k1 =  `

expected:   `{ k1 = "v1", k2 = "v2", k3 =  } `

### neovim version

```Text
v0.9.1
```

### Make sure you have done the following

- [X] I have updated to the latest version of the plugin.

**Comment by chrisgrieser** [comment]
yeah, that's a limitation due to the fact that this plugin uses pattern matching instead of treesitter to determine the text objects. And tressitter-textobjects are a non-goal for this plugin, since [treesitter-textobjs](https://github.com/nvim-treesitter/nvim-treesitter-textobjects) already does them.

The only reason this plugin nonetheless offers the inferior pattern-based value text objects is due to the fact that the treesitter-based value text objects isn't fully done yet (see #20). Hence, this is not an issue this plugin will/can fix, instead I'd indicate more interest for such text objects to the treesitter-textobj people.
